address/list.js

```diff
@@ -0,0 +1,96 @@
+var addr_default = [5,6,7,8];
+
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var url_host1=window.location.origin+"/"+split_url[1];//+"/"+split_url[2];
+
+$(document).ready(function () {
+    $('[data-toggle="tooltip"]').tooltip();
+    $('#create').on('click', function () {
+        $('.show-create').html('Create Service Request !');
+        $('.modal-create').modal('show');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8393720)
```suggestion
        $(".modal-create").modal('show');
```